packages/react/src/components/Code/index.tsx

```diff
@@ -64,17 +74,29 @@ const Code: React.ComponentType<React.PropsWithChildren<Props>> = ({
   }, [scrollable]);
 
   return (
-    <Highlighter
-      {...props}
-      PreTag={PreWithRef}
-      useInlineStyles={false}
-      className={classNames('Code', className, {
-        'Code--scrollable': scrollable
-      })}
-      tabIndex={scrollableRegion ? 0 : undefined}
-    >
-      {children}
-    </Highlighter>
+    <>
+      {(label || allowCopy) && (
+        <div className="Code__Header">
+          {label && (
+            <div>
+              <span id={`${id}-label`}>{label}</span>
+            </div>
```

Comment:
The extra div is not necessary.

```suggestion
            <span id={`${id}-label`}>{label}</span>
```